test/plugin/default.nix

```diff
@@ -6,6 +6,9 @@ let
   project = project' {
     inherit compiler-nix-name evalPackages;
     src = testSrc "plugin";
+    modules = [{
+      reinstallableLibGhc = builtins.compareVersions buildPackages.haskell-nix.compiler.${compiler-nix-name}.version "9.8.1" < 0;
```

Comment:
Is there a way to make this automatic, e.g. as part of comp-builder.nix or similar? Otherwise any project building with 9.8.1. & plugins will have this same problem. (Apologies for commenting on a merged PR; I only thought of this over the weekend ...)